MBFingerTipWindow.swift

```diff
@@ -0,0 +1,268 @@
+//
+//  MBFingerTipWindow.swift
+//
+//  Created by Felix Traxler on 05.08.19.
+//
+
+import Foundation
+import UIKit
+
+class MBFingerTipView: UIImageView {
```

Comment:
Swift modules almost act as namespaces, so we don't need to prefix classes in Swift, and we are trying to move away from 2 letter prefixes for Obj-C classes because MB can clash with Apple’s MobileBackup framework. Would you mind changing this to
```swift
@objc(MBXFingerTipView)
class FingerTipView
```

Reply:
Do I need the

`@objc(MBXFingerTipView)`

-prefix for every class in the swift file? (Sorry I am just a junior developer, but I really appreciate every feedback I can get)

Reply:
Public classes and methods that you want to expose to Objective-C. Which got me thinking, the `FingerTipClass` should probably be open to allow subclassing.